README.md

```diff
@@ -1,2 +1,261 @@
 # operator-certification-operator
-A Kubernetes operator to provision the OpenShift Operator Pipeline.
+A Kubernetes operator to provision resources for the operator certification
```

Comment:
This feels like an incomplete sentence. Maybe change to:

"A Kubernetes operator to provision resources for the operator certification pipeline."